src/updater/updater.cpp

```diff
@@ -112,9 +114,6 @@ void checkWeightsConsistency(
     extractor::EdgeBasedNodeDataContainer node_data;
     extractor::files::readNodeData(config.GetPath(".osrm.ebg_nodes"), node_data);
 
-    extractor::TurnDataContainer turn_data;
-    extractor::files::readTurnData(config.GetPath(".osrm.edges"), turn_data);
```

Comment:
Oh nice where is this simplification coming from?

Reply:
it is a left over from some other refactoring step. `turn_data` is loaded, but not used